packages/cascara/src/ui/Form/FormEmpty.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const propTypes = {};
+
+const temporaryEmptyStyles = {
+  alignItems: 'center',
+  display: 'flex',
+  height: '10em',
+  justifyContent: 'center',
+};
+
+const FormEmpty = () => (
+  <div className='ui placeholder segment' style={temporaryEmptyStyles}>
+    <em>No data...</em>
+  </div>
+);
+
+FormEmpty.propTypes = propTypes;
+
+export default FormEmpty;
```

Comment:
When these get renamed, I think we can place them inside /private so everyone can use them and we can make sure we are not exporting

Reply:
File removed